src/main/java/com/stripe/model/Invoice.java

```diff
@@ -42,8 +42,8 @@
   Long dueDate;
   Long endingBalance;
   Boolean forgiven;
-  @SerializedName("hosted_invoice_url") String hostedInvoiceURL;
-  @SerializedName("invoice_pdf") String invoicePDF;
+  String hostedInvoiceUrl;
+  String invoicePdf;
```

Comment:
Nice to see the `SerializedName`s going away.

Reply:
Yes! I was also able to remove a few manually defined accessors :)